.github/workflows/ci.yml

```diff
@@ -0,0 +1,141 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+name: Build flink-connector-elasticsearch
+on: [push, pull_request]
+jobs:
+  compile_ci_jdk8:
+    runs-on: ubuntu-latest
+    env:
+      MVN_GOALS: clean install
+      MVN_PROFILES: -Pcheck-convergence -Dscala-2.12
+      MVN_TEST_OPTIONS: -Dmaven.javadoc.skip=true -DskipTests -Dfast -U -B
+      MVN_COMMON_OPTIONS: -Dflink.forkCount=2 -Dflink.forkCountTestPackage=2
+      #      Temporary disable convergence phase since that will fail
+      #      MVN_COMMON_OPTIONS: -Dflink.convergence.phase=install -Dflink.forkCount=2 -Dflink.forkCountTestPackage=2
+      MVN_CONNECTION_OPTIONS: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Set JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+
+      - name: Set Maven 3.2.5
```

Comment:
Just FYI this restriction will likely no longer apply since shading should not occur in intermediate modules.

Reply:
But should we still set it to this specific version to be on the safe side?

Reply:
I'd match it with whatever version we expect contributors to use. Since the intent is to decouple the development of connectors & flink it seems reasonable to just use the latest maven version.
I think it's either that or 3.2.5 to stay compatible with the Flink development.

Reply:
I've discussed this offline with @AHeise and we decided to go for the last version (currently 3.8.5). I've updated it accordingly